lib/kafka_rest_client.js

```diff
@@ -327,6 +364,8 @@ KafkaRestClient.prototype.tryMakeRequest = function tryMakeRequest(produceMessag
     if (self.clientType === 'AtLeastOnce') {
         reqOpts.headers['Request-Type'] = 'sync';
     }
+    reqOpts.headers[self.lineageHeader] = self.applicationName;
+
```

Comment:
besides client id, we'd like to propagate client-version via header too so that it's easy tell what the client is using

Reply:
added in latest update.